### PR TITLE
Rabbit mgmt plugin

### PR DESCRIPTION
**Pull request by jdextercsc**
Adds the rabbitmq mgmt plugin support

### CODE REVIEW

**Comment by mtnbikenc** [comment]
You've got the commit for the nova stuff on this PR too.  Need to reset and drop that commit.

File: roles/rabbitmq/tasks/main.yml [hunk 1]
```diff
@@ -114,6 +114,17 @@
   run_once: true
   tags: rabbitmq_policy
 
+- name: enable plugins
+  command: rabbitmq-plugin enable {{ item }}
+  with_itmes:
```
Comment (mtnbikenc) [inline roles/rabbitmq/tasks/main.yml]: Typo with_items

**Comment by mtnbikenc** [comment]
Code review :+1:

**Comment by jcpowermac** [comment]
code review :+1:

**Pull request merged**